src/main/java/com/flytrap/rssreader/global/config/WebConfig.java

```diff
@@ -17,4 +18,15 @@ public class WebConfig implements WebMvcConfigurer {
     public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers) {
         resolvers.add(authorizationArgumentResolver);
     }
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**")
+            .allowedOrigins("http://localhost:5173")
+            .allowedMethods("GET", "POST", "PUT", "DELETE")
```

Comment:
Head, Option, Patch 가 추가로 되어야 합니다.

```suggestion
               .allowedMethods(
                       HttpMethod.OPTIONS.name(), HttpMethod.GET.name(),
                       HttpMethod.POST.name(), HttpMethod.PUT.name(),
                       HttpMethod.PATCH.name(), HttpMethod.DELETE.name(),
                       HttpMethod.HEAD.name()
               )                      
               .allowedHeaders(HttpHeaders.AUTHORIZATION, HttpHeaders.CONTENT_TYPE)

```